pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -44,11 +49,26 @@ func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *
 	// Do not fail, but continue after logging error as we can still dump flows with no timestamps.
 	sysctl.EnsureSysctlNetValue("netfilter/nf_conntrack_timestamp", 1)
 
-	return &connTrackSystem{
+	ct := &connTrackSystem{
 		nodeConfig,
 		serviceCIDR,
 		&netFilterConnTrack{},
 	}
+
+	// Set the antrea_agent_conntrack_max_connection_count prometheus metric
+	cmdOutput, err := exec.Command("cat", "/proc/sys/net/nf_conntrack_max").Output()
```

Comment:
could you use GetSysctlNet from pkg/agent/util/sysctl?

Reply:
Thanks, done.